### DISCUSSION TITLE
Working with wix-style-react

### DISCUSSION THREAD

**Discussion by ivanji** [vercel/next.js#14013] [Help]
Hi guys - anyone here has tried implementing NextJS with wix-style-react (https://github.com/wix/wix-style-react/)? This project main bundler is called Yoshi and also makes use of Stylable - it's been a bit of a pain to set up with Next. Wondering if anyone by any chance has had it implemented and could point me in the right direction with the Webpack config.

Some docs https://wix-style-react.now.sh/?path=/story/introduction-getting-started--usage-without-yoshi that seem outdated. Thanks.

**Comment by McZenith** [comment]
I'm currently also looking to use wix-style-react with Nextjs...
Have you been able to find a walk around?

**Reply by ivanji** [reply]
The guys @ Stylable have been looking into it as it ended up being more complex than it appears. Please see https://github.com/wix/stylable/issues/1212
